app/src/main/java/com/plan2meet/dto/ContactDTO.kt

```diff
@@ -8,9 +8,13 @@ class ContactDTO (
     @SerializedName("email") val email     : String = "",
     @SerializedName("phone") val phone     : String = "", // this value cannot be an Integer due to the size of "Int"
     @SerializedName("favorite") var favorite : Boolean = false)
+
+
 {
     @Override
     override fun toString(): String {
-        return "My name is $firstName $lastName, my e-mail is: $email and my phone # is: $phone"
+        //use variable to help clean up return code
+        val contact = "My name is $firstName $lastName, my e-mail is: $email and my phone # is: $phone"
+        return contact
```

Comment:
I'm neutral on this.  Introducing a variable for something used only once does not decrease technical debt.